app/src/data_loader/trade_loader/websocket_utility.py

```diff
@@ -0,0 +1,35 @@
+import asyncio
+import json
+import logging
+import websockets
+
+class WebSocketUtility:
+    __instance = None
+    __should_send = False
+    __message = None
+    Users = set()
+    WebSocketDict = {}
+    
+    @staticmethod
+    def getInstance():
+        if WebSocketUtility.__instance == None:
+            WebSocketUtility()
+        return WebSocketUtility.__instance
+    def __init__(self):
+        """ Virtually private constructor. """
+        if WebSocketUtility.__instance != None:
+            raise Exception("This class is a singleton!")
+        else:
+            WebSocketUtility.__instance = self
+    
+    def set_send_status(self, flag):
+        self.__should_send = flag
+        
+    def get_send_status(self):
+        return self.__should_send
+    
+    def set_stock_list(self, msg):
```

Comment:
In python it is better to write variable names in a way that they represent their type and purpose. I don't understand whether `msg` is list or it is an string. if it must be a stock list please name it `stock_list`

Reply:
Yes you're right. I think I should rename it to `stock_signaled`; because in Angular mode we don't want to send list of signals. Instead, every single stock signaled should send immediately. What's your opinion?

Reply:
I aggre